ramls/age-to-lost-background-processes.raml

```diff
@@ -0,0 +1,31 @@
+#%RAML 1.0
+title: Age to lost background processes
+version: v0.1
+protocols: [ HTTP, HTTPS ]
+baseUri: http://localhost:9130
+
+documentation:
+  - title: Background processes for ageing borrowed items to lost
+    content: <b>Background processes for ageing borrowed items to lost</b>
+
+/circulation/scheduled-age-to-lost:
+  post:
+    responses:
+      204:
+        description: "The process completed successfully"
```

Comment:
Do we need to add statuses for bad inputs?  For example, if the job can only set to be run daily or weekly, and someone tries to send a post request for "monthly?"  Or is the idea that we wait on those until we have a better sense of exactly what the structure of the post request will look like?

Reply:
> Do we need to add statuses for bad inputs? 

Good question. I'll check the code in case some are missing.

> For example, if the job can only set to be run daily or weekly, and someone tries to send a post request for "monthly?" Or is the idea that we wait on those until we have a better sense of exactly what the structure of the post request will look like?

As this change means that the scheduling becomes entirely external (for the moment), that particular example isn't a concern for this change. It may well be a concern for the longer term approach we take and I should mention it in the design proposal.